assets/styles/authoring/model-editors.scss

```diff
@@ -16,13 +16,19 @@
       word-wrap: break-word;
       overflow: visible;
 
+      td {
```

Comment:
This moves that table dropdown button into the corner of the cell instead of the corner of the table. It seemed like a nice visual improvement to me.